test/e2e/readme.md

```diff
@@ -2,12 +2,14 @@
 
 Celestia uses the [knuu](https://github.com/celestiaorg/knuu) framework to orchestrate clusters of nodes in a network for end to end testing. This relies on Docker and a kubeconfig (in `~/.kube/config`) to access the Kubernetes cluster.
 
+End to end tests pull docker images from ghcr.io/celestiaorg/celestia-app. These are automatically published when tagging a new release or when opening a pull request. If you wish to manually test a specific commit, you can manually publish the image by first running `make build-ghcr-docker` (from the root directory) and then running `make publish-ghcr-docker`. You must have permission to push to the ghcr.io/celestiaorg/celestia-app repository.
```

Comment:
> from ghcr.io/celestiaorg/celestia-app

[optional] Should we add a link somewhere in this paragraph to: https://github.com/celestiaorg/celestia-app/pkgs/container/celestia-app

so that developers know where to check after they have run `publish-ghcr-docker` to verify that the image they built and pushed locally made it to ghcr?

Reply:
If you go to ghcr.io/celestiaorg/celestia-app it should redirect you to that sight so you can confirm that it  is published

Reply:
Yea it does redirect. Can resolve.